addon/services/hover-intent.js

```diff
@@ -0,0 +1,126 @@
+import Ember from 'ember';
```

Comment:
Could we make this an implicit private service by prefixing this with a `-`. That way, we prevent this from polluting the global app namespace.